pkg/mapfs/fs_test.go

```diff
@@ -478,3 +478,26 @@ func TestFS_RemoveAll(t *testing.T) {
 		require.ErrorIs(t, err, fs.ErrNotExist)
 	})
 }
+
+func TestFS_WithUnderlyingRoot(t *testing.T) {
+	root := "testdata/subdir"
```

Comment:
Is there any reason why `testdata` is used? Can we use `filepath.Join(t.TempDir(), "subdir"`?

Reply:
The `testdata/subdir` contains some files to which access is tested.

Reply:
OK, thanks.